signing/signer.go

```diff
@@ -13,9 +13,20 @@ import (
 	"github.com/spacemeshos/go-spacemesh/log"
 )
 
+type Domain byte
+
+const (
+	Default Domain = iota
```

Comment:
default domain? this is for stuff that is not refactored? there should be no such thing